README.rst

```diff
@@ -48,21 +50,47 @@ Install ``scikit-hep`` like any other Python package:
 
 .. code-block:: bash
 
-   pip install scikit-hep
+   python -m pip install scikit-hep
 
-or similar (use ``virtualenv``, etc. if you wish).
+or similar (use ``venv``, etc. if you wish).
 
-Strict dependencies
-...................
+Package version and dependencies
+................................
 
 Please check the ``setup.py`` and ``requirements.txt`` files for the list
 of Python versions supported and the list of Scikit-HEP project packages
 and dependencies included, respectively.
 
-Documentation
-.............
-
-The project documentation is hosted on the `Scikit-HEP website`_.
-
-
-.. _Scikit-HEP website: http://scikit-hep.org/
+For any installed `scikit-hep` the following displays the actual versions
+of all Scikit-HEP dependent packages installed, for example:
+
+.. code-block:: python
+
+>>> import skhep
+>>> skhep.show_versions()
+
+System:
+    python: 3.8.6 (default, Sep 24 2020, 21:45:12)  [GCC 8.3.0]
+executable: /usr/local/bin/python
+   machine: Linux-4.19.104-microsoft-standard-x86_64-with-glibc2.2.5
+
+Python dependencies:
+       pip: 20.2.3
+setuptools: 50.3.0
+     numpy: 1.19.2
+     scipy: 1.5.2
+    pandas: 1.1.2
+matplotlib: 3.3.2
+
+Scikit-HEP package version and dependencies:
+        awkward: 0.13.0
+boost_histogram: 0.11.0
+  decaylanguage: 0.8.0
+       hepunits: 2.0.0
+           hist: 2.0.0
+        iminuit: 1.5.2
+         mplhep: 0.2.2
+       particle: 0.12.0
+          skhep: 1.0.0
+ uproot_methods: 0.7.4
+         uproot: 3.12.0
```

Comment:
I take it you are going to manually maintain the output of this in the readme?

Also, shouldn't this whole block be indented?

Reply:
I do not intend to change it often at all, no. At some point we can remove the output. For the beginning of the metapackage I find it important to be a bit more verbose, not the least because this little function is handy.

Ah, I copied from my prompt and indeed it should be indented. I will correct that.

Reply:
I'm resolving. Let me know if there's anything else. Otherwise we're good. Thanks.